src/style.css

```diff
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  display: block;
+  margin: 8px;
+}
+
+.form {
+  display: flex;
+  justify-content: left;
+  margin: 20px auto 0;
+}
+
+.bigform {
+  width: 100%;
```

Comment:
move this property to .form selector

Reply:
removed from .bigform & added to -> .form